### PR TITLE
test: Add K8s test to check quality of service for pods

### LINKED ISSUE #972: Add K8s test to check quality of service for pods
Add a test that configure Pods with an assigned particular Quality of Service (QoS) class.

### PR DESCRIPTION
**Pull request by GabyCT**
Add a test that configure Pods with an assigned particular
Quality of Service (QoS) class.

Fixes #972

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/kubernetes/untrusted_workloads/pod-besteffort.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  annotations:
+    io.kubernetes.cri-o.TrustedSandbox: "false"
+    io.kubernetes.cri.untrusted-workload: "true"
+spec:
+  containers:
+  - name: qos-besteffort
+    image: busybox
+    command:
+        - sleep
+        - "30"
```
Comment (jodh-intel) [inline integration/kubernetes/untrusted_workloads/pod-besteffort.yaml]: Might be safer to use the `tail -f /dev/null` trick?

Comment (jodh-intel) [reply integration/kubernetes/untrusted_workloads/pod-besteffort.yaml]: Here and below.

**Comment by GabyCT** [comment]
@jodh-intel changes applied

**Comment by GabyCT** [comment]
@jodh-intel changes applied

**Comment by jodh-intel** [comment]
Thanks @GabyCT! Still...

lgtm

**Comment by jodh-intel** [comment]
Thanks @GabyCT! Still...

lgtm

**Comment by jodh-intel** [comment]
Thanks @GabyCT! Still...

lgtm

**Comment by jodh-intel** [comment]
Thanks @GabyCT! Still...

lgtm

**Comment by jodh-intel** [comment]
It seems to me that github should be making these updates *transactional* - I got errors every time for the 4 identical comments above!!

**Comment by jodh-intel** [comment]
Oh - the irony - more errors...

**Comment by jodh-intel** [comment]
> You can't comment at this time.

Actually, it seems I can! If only ATMs were like this... ;)

**Pull request merged**